src/data/data.ts

```diff
@@ -388,14 +388,12 @@ export const Vaccines: { [key: string]: VaccineValue } = {
   },
 }
 
-export const TheirVaccine = {
-  vaccinated: {
-    label: i18n.t('data.their_vaccine.yes'),
-  },
-  unvaccinated: {
-    label: i18n.t('data.their_vaccine.no'),
-  },
-  undefined: {
-    label: i18n.t('data.their_vaccine.unknown'),
-  },
+const yesVaccinatedDesc = i18n.t('data.yes_vaccinated')
+const noVaccinatedDesc = i18n.t('data.no_vaccinated')
+const unknownVaccinatedDesc = i18n.t('data.unknown_vaccinated')
+
+export const TheirVaccine: { [key: string]: CheckBoxFormValue } = {
+  vaccinated: segmentedFormValue(i18n.t('data.their_vaccine.yes'), yesVaccinatedDesc, 1 / 2),
```

Comment:
Is this accurate? I thought other discussions had it all the way down to .2-.16

Reply:
This is a multiplier tedmk found for transmission from vaccinated people who are known to have covid, which is different from the risk reduction from the vaccine.

Reply:
@beshaya but if, in a given area, a small percentage of people have COVID, wouldn't the absolute risk be far lower than 1/2?